app/components-react/windows/go-live/GoLiveSettings.tsx

```diff
@@ -29,11 +29,11 @@ export default function GoLiveSettings() {
     isAdvancedMode,
     protectedModeEnabled,
     error,
-    canAddDestinations,
     isLoading,
-  } = useGoLiveSettings(view => {
-    const linkedPlatforms = view.linkedPlatforms;
-    const customDestinations = view.customDestinations;
+    canAddDestinations,
+  } = useGoLiveSettings().selectExtra(module => {
+    const linkedPlatforms = module.linkedPlatforms;
+    const customDestinations = module.customDestinations;
```

Comment:
I was unable to describe types for `select` in the way to support `selectExtra` syntax. Because of 2 tsconfigs I have weird issues here. I'll fix that when we switch to a single tsconfig